ts/features/design-system/core/DSListItems.tsx

```diff
@@ -22,7 +22,7 @@ import { ProductCategoryEnum } from "../../../../definitions/cgn/merchants/Produ
 import CgnMerchantListItem from "../../bonus/cgn/components/merchants/CgnMerchantListItem";
 import DetailedlistItemComponent from "../../../components/DetailedlistItemComponent";
 import { TimelineOperationListItem } from "../../idpay/initiative/details/components/TimelineOperationListItem";
-import { OperationTypeEnum } from "../../../../definitions/idpay/timeline/TransactionOperationDTO";
+import { OperationTypeEnum } from "../../../../definitions/idpay/TransactionOperationDTO";
```

Comment:
Just for reference. Is is not good to have the design system to depends on idpay or other specific features...

Reply:
cc @dmnplb